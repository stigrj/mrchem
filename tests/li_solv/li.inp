{
"world_prec": 1.0e-3,
"world_size": 5,
"MPI": {
  "numerically_exact": true
},
"Molecule": {
  "charge": 1,
  "coords": "Li 0.0 0.0 0.0"
},
"WaveFunction": {
  "method": "pbe0",
  "environment": "pcm"
},
"PCM": {
  "SCRF": {
    "kain": 5,
    "max_iter": 100,
    "dynamic_thrs": true,
    "optimizer": "potential"
  },
  "Cavity": {
    "sigma": 0.5,
    "spheres": "0.0 0.0 0.0 4.0"
  },
  "Permittivity": {
    "epsilon_in": 1.0,
    "epsilon_out": 2.0,
    "formulation": "exponential"
  }
},
"SCF": {
  "run": false,
  "guess_type": "sad_dz"
}
}
