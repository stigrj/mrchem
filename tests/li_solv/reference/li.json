{
  "input": {
    "constants": {
      "angstrom2bohrs": 1.8897261246257702,
      "dipmom_au2debye": 2.5417464739297717,
      "electron_g_factor": -2.00231930436256,
      "fine_structure_constant": 0.0072973525693,
      "hartree2ev": 27.211386245988,
      "hartree2kcalmol": 627.5094740630558,
      "hartree2kjmol": 2625.4996394798254,
      "hartree2simagnetizability": 78.9451185,
      "hartree2wavenumbers": 219474.6313632,
      "light_speed": 137.035999084
    },
    "molecule": {
      "cavity": {
        "sigma": 0.5,
        "spheres": [
          {
            "center": [
              0.0,
              0.0,
              0.0
            ],
            "radius": 4.0
          }
        ]
      },
      "charge": 1,
      "coords": [
        {
          "atom": "li",
          "xyz": [
            0.0,
            0.0,
            0.0
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": -1,
      "numerically_exact": true,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 5,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -4
    },
    "printer": {
      "file_name": "li",
      "print_constants": false,
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 0.001,
          "shared_memory": false
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 0.001
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 0.001,
          "shared_memory": false,
          "smooth_prec": 0.001
        },
        "reaction_operator": {
          "density_type": "total",
          "dynamic_thrs": true,
          "epsilon_in": 1.0,
          "epsilon_out": 2.0,
          "formulation": "exponential",
          "kain": 5,
          "max_iter": 100,
          "optimizer": "potential",
          "poisson_prec": 0.001
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe0"
              }
            ],
            "spin": false
          }
        }
      },
      "initial_guess": {
        "environment": "PCM",
        "external_field": "None",
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": false,
        "method": "DFT (PBE0)",
        "prec": 0.001,
        "relativity": "None",
        "restricted": true,
        "screen": 12.0,
        "type": "sad",
        "zeta": 2
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 0.001,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        0.0
      ],
      "charge": 1,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 7.201515824859437e-14,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            0.0,
            0.0,
            0.0
          ],
          "vector_el": [
            0.0,
            0.0,
            0.0
          ],
          "vector_nuc": [
            0.0,
            0.0,
            0.0
          ]
        }
      },
      "geometry": [
        {
          "symbol": "Li",
          "xyz": [
            0.0,
            0.0,
            0.0
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -2.2040278701953486
        ],
        "occupation": [
          2.0
        ],
        "spin": [
          "p"
        ],
        "sum_occupied": -4.408055740390697
      },
      "scf_energy": {
        "E_ee": 3.332799383894309,
        "E_eext": 0.0,
        "E_el": -7.094583823998505,
        "E_en": -16.53075528769529,
        "E_kin": 7.675564301900199,
        "E_next": 0.0,
        "E_nn": 0.0,
        "E_nuc": -0.1921541203648782,
        "E_tot": -7.2867379443633835,
        "E_x": -0.41648449531759685,
        "E_xc": -1.2837729977609404,
        "Er_el": 0.12806527098081377,
        "Er_nuc": -0.1921541203648782,
        "Er_tot": -0.06408884938406478
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 1,
      "nthreads": 1,
      "routine": "mrchem.x",
      "total_cores": 1,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 3.332799383894309,
        "E_eext": 0.0,
        "E_el": -7.094583823998505,
        "E_en": -16.53075528769529,
        "E_kin": 7.675564301900199,
        "E_next": 0.0,
        "E_nn": 0.0,
        "E_nuc": -0.1921541203648782,
        "E_tot": -7.2867379443633835,
        "E_x": -0.41648449531759685,
        "E_xc": -1.2837729977609404,
        "Er_el": 0.12806527098081377,
        "Er_nuc": -0.1921541203648782,
        "Er_tot": -0.06408884938406478
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
