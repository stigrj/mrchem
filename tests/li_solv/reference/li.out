                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         solvent-input                            ***
***         Git commit hash    685b3596305e75a76eff-dirty               ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Wed Jul 13 16:03:41 2022 +0200           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               1
 Total cores             :                                               1
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : OpenMP (1 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   16.00000    16.00000    16.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -16.00000   -16.00000   -16.00000 ]
 upper bounds          : [   16.00000    16.00000    16.00000 ]
 total length          : [   32.00000    32.00000    32.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               1
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Li            :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                             Solvation Cavity
---------------------------------------------------------------------------
 Formulation             :                                     exponential
 Cavity width            :                                        0.500000
 Dielectric constant     :            (in)                        1.000000
                         :           (out)                        2.000000
---------------------------------------------------------------------------
    N      Radius        :               x               y               z
---------------------------------------------------------------------------
    0    4.000000        :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                Computing Initial Guess Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-03
 Screening               : 1.20000e+01 StdDev
 Restricted              : True
 Functional              : LDA (SVWN5)
 AO basis                : Hydrogenic orbitals
 Zeta quality            : 2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              1.000000000000e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Environment             : PCM
 External fields         : None
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  7.675564301900
 E-N energy              :            (au)                -16.530755287695
 Coulomb energy          :            (au)                  3.332799383894
 Exchange energy         :            (au)                 -0.416484495318
 X-C energy              :            (au)                 -1.283772997761
 N-N energy              :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Reaction energy (el)    :            (au)                  0.128065270981
 Reaction energy (nuc)   :            (au)                 -0.192154120365
 Reaction energy (tot)   :            (au)                 -0.064088849384
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -7.094583823999
 Nuclear energy          :            (au)                 -0.192154120365
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.286737944363e+00
                         :      (kcal/mol)             -4.572497095103e+03
                         :        (kJ/mol)             -1.913132784591e+04
                         :            (eV)             -1.982822406774e+02
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -2.204027870195
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -4.408055740391
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               1
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Li            :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  7.675564301900
 E-N energy              :            (au)                -16.530755287695
 Coulomb energy          :            (au)                  3.332799383894
 Exchange energy         :            (au)                 -0.416484495318
 X-C energy              :            (au)                 -1.283772997761
 N-N energy              :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Reaction energy (el)    :            (au)                  0.128065270981
 Reaction energy (nuc)   :            (au)                 -0.192154120365
 Reaction energy (tot)   :            (au)                 -0.064088849384
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -7.094583823999
 Nuclear energy          :            (au)                 -0.192154120365
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.286737944363e+00
                         :      (kcal/mol)             -4.572497095103e+03
                         :        (kJ/mol)             -1.913132784591e+04
                         :            (eV)             -1.982822406774e+02
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -2.204027870195
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -4.408055740391
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000       -0.000000       -0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :        0.000000        0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m 51s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
